### PR TITLE
Broken cancel button in InstructorStudentRecords comment edit #3733

### LINKED ISSUE #3733: Broken cancel button in InstructorStudentRecords comment edit
isrcommenteditcancel
The comment editing section is not removed. I am to blame because this is introduced when I was refining the slice:
explained

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #3733

### CODE REVIEW

**Comment by damithc** [comment]
Can we add a test to cover this?

**Comment by wkurniawan07** [comment]
Should be able to.

**Comment by wkurniawan07** [comment]
Test added. Fails before the fix, passes after.

File: src/test/java/teammates/test/pageobjects/InstructorStudentRecordsPage.java [hunk 1]
```diff
@@ -80,11 +81,25 @@ public boolean verifyEditCommentButtonClick(int id) {
...
+        assertFalse(isElementVisible(By.id("commentTextEdit" + id)));
+    }
 
     private WebElement getCommentEditLink(int id) {
         return browser.driver.findElement(By.id("commentedit-" + id));
     }
 
+    private WebElement getCommentEditCancelLink(int id) {
+        return browser.driver.findElement(By.id("commentsave-" + id))
+                             .findElement(By.xpath("./following-sibling::input"));
```
Comment (bamboo3250) [inline src/test/java/teammates/test/pageobjects/InstructorStudentRecordsPage.java]: This is an assumption that the Cancel button must be next to the Save button. At the moment, the Cancel button doesn't have an ID so I recommend to give it an ID later if possible. For now, it is good enough for a quick fix.

**Pull request merged**